banyand/measure/measure_write.go

```diff
@@ -131,9 +131,12 @@ func (s *measure) write(shardID common.ShardID, entity []byte, entityValues tsdb
 	}
 	s.indexWriter.Write(m)
 	if s.processorManager != nil {
-		s.processorManager.onMeasureWrite(&measurev1.WriteRequest{
-			Metadata:  s.GetMetadata(),
-			DataPoint: value,
+		s.processorManager.onMeasureWrite(&measurev1.InternalWriteRequest{
+			Request: &measurev1.WriteRequest{
+				Metadata:  s.GetMetadata(),
+				DataPoint: value,
+			},
+			EntityValues: entityValues[1:],
```

Comment:
Why do you remove the first value? This value is from the measure's specification. The first is not a reserved entity.

Reply:
> Why do you remove the first value? This value is from the measure's specification. The first is not a reserved entity.

https://github.com/apache/skywalking-banyandb/blob/e68249a385a720d075eab52b496d7c4b4679ba41/pkg/partition/entity.go#L58-L59

The first entity is just the source measure name